analytics/index.yaml

```diff
@@ -254,9 +252,10 @@ paths:
           example: PhilipFry
         - name: cid
           in: query
-          #TODO: What's the difference between _uid, cid and uid parameters? Are some of those deprecated?
           description: |
-            Visitor ID.
+            This parameter overwrites visitors `config_id`. For further explanation please read following article:
```

Comment:
I'm trying to give these information parameter description (in this sequence):

* full name of the parameter used by other parts of the documentation. Ideally user should be able to refer to concept represented by parameter using it and search for it in docs or help site. Example: `Configuration ID` or `Fingerprint ID`.
* Short description of the concept. It doesn't have to be extensive, just a summary with a link to full article will do. Example: `semi-unique hash generated for Visitor's client (based on configuration and installed plugins)`.
* Format of the parameter (is it's too complicated to describe with OpenAPI options) 

Can you reformat description to add these information's?